pkg/util/file_indexer.go

```diff
@@ -276,3 +266,13 @@ func write(filePath string, fi *FileIndex) error {
 	// 0600 is the mask used when a file is created using os.Create hence defaulting
 	return ioutil.WriteFile(filePath, jsonData, 0600)
 }
+
+// writes the map of walked files and info about them, in a file
+// filepath is the location of the file to which it is supposed to be written
```

Comment:
I am lost over here which `filepath` you are talking about?

Reply:
should be filePath .. sorry